managed-services/transactional-email/how-to/generate-api-keys-for-tem-with-iam.mdx

```diff
@@ -18,7 +18,7 @@ This page shows you how to generate [API keys](/identity-and-access-management/i
 - [Owner](/identity-and-access-management/iam/concepts/#owner) status or [IAM permissions](/identity-and-access-management/iam/concepts/#permission) allowing you to perform actions in the intended Organization
 - A domain name
 - [Configured your domain with Transactional Email](/managed-services/transactional-email/how-to/configure-domain-with-transactional-email)
-- Added [SPF, DKIM](/managed-services/transactional-email/how-to/add-spf-dkim-records-to-your-domain) and [MX](/managed-services/transactional-email/how-to/add-mx-record-to-your-domain/) records to your domain
+- Added [SPF[, DKIM](/managed-services/transactional-email/how-to/add-spf-dkim-records-to-your-domain), and [MX](/managed-services/transactional-email/how-to/add-mx-record-to-your-domain/) records to your domain
```

Comment:
SPF and DKIM configuration are on the same doc page, did you split them on purpose?